models/learner.py

```diff
@@ -247,333 +275,465 @@ def loadSavedModel(log_folder, valid_models, cuda=True):
+        dataloader_valid = torch.utils.data.DataLoader(valid_set, **data_loader_params)
+        dataloader_test  = torch.utils.data.DataLoader(test_set, **data_loader_params_test)
+        # ------- Load ground truth states/ target positions (for plots and GTC) -------
+
+        # ========================= Print some info =========================
         # Stats about actions
         action_set = set(actions)
         n_actions = int(np.max(actions) + 1)
         print("{} unique actions / {} actions".format(len(action_set), n_actions))
         n_pairs_per_action = np.zeros(n_actions, dtype=np.int64)
         n_obs_per_action = np.zeros(n_actions, dtype=np.int64)
-
         for i in range(n_actions):
             n_obs_per_action[i] = np.sum(actions == i)
-
         print("Number of observations per action")
         print(n_obs_per_action)
-
-        dissimilar_pairs, same_actions_pairs = None, None
-        if not self.no_priors:
-            dissimilar_pairs, same_actions_pairs = findPriorsPairs(self.batch_size, minibatchlist, actions, rewards,
-                                                                   n_actions, n_pairs_per_action)
-
-        if self.use_vae and self.perceptual_similarity_loss and self.path_to_dae is not None:
-
-            self.denoiser = SRLModules(state_dim=self.state_dim_dae, action_dim=self.dim_action,
-                                       model_type="custom_cnn",
-                                       cuda=self.cuda, losses=["dae"])
-            self.denoiser.load_state_dict(th.load(self.path_to_dae))
-            self.denoiser.eval()
-            self.denoiser = self.denoiser.to(self.device)
-            for param in self.denoiser.parameters():
-                param.requires_grad = False
-
-        if self.episode_prior:
-            idx_to_episode = {idx: episode_idx for idx, episode_idx in enumerate(np.cumsum(episode_starts))}
-            minibatch_episodes = [[idx_to_episode[i] for i in minibatch] for minibatch in minibatchlist]
-
-        data_loader = DataLoader(minibatchlist, images_path, n_workers=N_WORKERS, multi_view=self.multi_view,
-                                 use_triplets=self.use_triplets, is_training=True, apply_occlusion=self.use_dae,
-                                 occlusion_percentage=self.occlusion_percentage)
-        test_data_loader = DataLoader(test_minibatchlist, images_path, n_workers=N_WORKERS, multi_view=self.multi_view,
-                                      use_triplets=self.use_triplets, max_queue_len=1, is_training=False,
-                                      apply_occlusion=self.use_dae, occlusion_percentage=self.occlusion_percentage)
+        print("Train: {} minibatches, {} samples".format(len(dataloader_train), len(train_set)))
+        print("Valid: {} minibatches, {} samples".format(len(dataloader_valid), len(valid_set)))
+        # =======================================================================
```

Comment:
Construction site...I hope that you plan to make your loader complatible with the following section :)